fancy_toys.jl

```diff
@@ -0,0 +1,34 @@
+# This is a collection of toys under the Yggdrasil tree for the good <s>kids</s>
+# developers.  These utilities can be employed in builder files.
+
+using BinaryBuilder
+
+"""
+    run_platform(platform) -> Bool
+
+Return whether the builder for the given `platform` should be run.
+
+This is useful when the builder has different platform-dependent elements
+(sources, script, products, etc...) that make it hard to have a single
+`build_tarballs` call.
+"""
+function run_platform(platform)
```

Comment:
I like to make "question verbs" like this have a question word in them.  E.g. `should_build_platform()`; let's do that instead of `run_platform()`